### PR TITLE
Add missing ios_smoke tests to IOS colletion

### PR DESCRIPTION
**Pull request by justjais**
Depends-On: https://github.com/ansible-collections/cisco.ios/pull/82
##### SUMMARY

Add missing ios_smoke tests to IOS colletion

##### COMPONENT NAME

ios_smoke

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

File: tests/integration/targets/ios_smoke/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+- { include: cli.yaml, tags: ['cli'] }
```
Comment (GomathiselviS) [inline tests/integration/targets/ios_smoke/tasks/main.yaml:2]:
```suggestion
- { include: cli.yaml, tags: ['network_cli'] }
```

**Comment by justjais** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**